kernel/kernel-config-server/README.md

```diff
@@ -78,38 +96,39 @@ server.port = 51000
 #adding context path
 server.servlet.path=/config
 
-# Uncomment spring.cloud.config.server.git.uri and spring.cloud.config.server.git.search-paths for # connecting to git Repo for configuration.
-#################################################################
-#Git repository location where configuration files are stored
-#spring.cloud.config.server.git.uri=<your-git-repository-URL>
+spring.profiles.active=composite
 
-#Path inside the GIT repo where config files are stored, in our case they are inside config directory
-#spring.cloud.config.server.git.search-paths=<folder-in-git-repository-containing-configuration>
+#Server would return a HTTP 404 status, if the application is not found.By default, this flag is set to true.
+spring.cloud.config.server.accept-empty=false
 
-# Uncomment spring.profiles.active and spring.cloud.config.server.native.search-locations for     # connecting to local file system for configuration.
-#################################################################
-# spring.profiles.active=native
+## As spring.profiles.active is composite, use env variable to provide values for git configuration as below
+##########################
+##Git repository location where configuration files are stored
+# SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_URI=<your-git-repository-URL>
 
-# spring.cloud.config.server.native.search-locations=file:///<config-location-on-your-system>
+##Type of repository, possible types are git, svn, native
+# SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_TYPE=git
 
-#Server would return a HTTP 404 status, if the application is not found.By default, this flag is set to true.
-spring.cloud.config.server.accept-empty=false
+##Branch/label to refer for in config repository
+# SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_DEFAULT_LABEL=<your-git-repository-branch>
 
-#Spring Cloud Config Server makes a clone of the remote git repository and if somehow the local copy gets 
+#Spring Cloud Config Server makes a clone of the remote git repository and if somehow the local copy gets
 #dirty (e.g. folder content changes by OS process) so Spring Cloud Config Server cannot update the local copy
 #from remote repository. For Force-pull in such case, we are setting the flag to true.
-spring.cloud.config.server.git.force-pull=true
-
-# Disabling health endpoints to improve performance of config server while in development, can be commented out in production.
-health.config.enabled=false
+# SPRING_CLOUD_CONFIG_SERVER_COMPOSITE_0_FORCE_PULL=true
 
-# Setting up refresh rate to 1 minute so that config server will check for updates in Git repo after every one minute,
+# Setting up refresh rate to 5 seconds so that config server will check for updates in Git repo after every one minute,
```

Comment:
5 secopnds, every one min ?

Reply:
We can mention 1 minute. I changed it to 5 secs because actual configuration I could see in existing properties as 5 seconds https://github.com/mosip/commons/blob/develop/kernel/kernel-config-server/src/main/resources/bootstrap.properties#L44

Reply:
No, I was saying the text is sounding right, we have 5 seconds, but every on e minute ?
"Setting up refresh rate to 5 seconds so that config server will check for updates in Git repo after every one minute,"

Reply:
Ah, correcting it